README.md

```diff
@@ -1,3 +1,5 @@
 # test-01
 
 jhhjkjhjkhkjs
+
+;wjkjewhkjeh
```

Comment:
Please remove the last line that has a random string of characters. It seems to be unintentional and doesn't contribute to the content.